repos/omeka-i18n-module/src/Controller/TranslationSyncController.php

```diff
@@ -40,12 +49,15 @@ public function showAction()
         return (new ViewModel(
             [
                 'form' => $form,
+                'isEnabled' => $this->isEnabled,
             ]
         ))->setTemplate('admin/i18n-synchronize/show');
     }
 
     public function processAction()
     {
+        if (!$this->isEnabled) return null;
```

Comment:
Hm, the typical approach with feature flagged web actions/controllers is to conditionally modify the routing tree so the controller doesn't need to be aware of the flag. That also helps identify places that are incorrectly linking to it (because generating a link to that route will now throw an exception).

Reply:
Been mulling over this one, I can't think of a straight nice way of doing this at the moment. The module is currently setting dynamic routes in `Module.php:151` but doesn't yet have access to the Omeka settings (from the database). Similar story with the navigation, it's all out of order.

I'm going to leave this as it is, but I do want to come back to this as a problem in general. In Omeka, the level of granularity of a feature flag is generally encapsulated in a module. So it may be the case that we split out the Transifex integration into its own module.